### ISSUE TITLE
Publish Ambry to Maven Central

### ISSUE THREAD

**Issue by eolivelli** [linkedin/ambry#418]
Hi, I would like to embed ambry in my (distributed) application. I use Maven, would it be possible to publish Ambry artifacts to Maven Central ?

**Comment by vgkholla** [comment]
Sorry for the late reply. I'll take a look at how this can be done.

**Comment by vgkholla** [comment]
We will probably release a version once the `NonBlockingRouter` is stable (another month or so). Does that work for you?

**Comment by eolivelli** [comment]
Sounds good to me.
Thanks

**Comment by heipacker** [comment]
hi, when NonBlockingRouter can stable?

**Comment by heipacker** [comment]
hi @vgkholla, when NonBlockingRouter can stable?

**Comment by pnarayanan** [comment]
@heipacker  It is stable in the current version, have not cut a release yet but we are good to do that now. Are you looking for artifacts in Maven?

**Comment by heipacker** [comment]
no, we need a object store system in work, so we just want to know this a  stable version

**Comment by sriramsub** [comment]
It definitely is. It is running in production at Linkedin.

**Comment by eolivelli** [comment]
@pnarayanan any news on this topic ?
I'm looking forward an official release on Maven or a way to have Maven 'official' artifacts
